### DISCUSSION TITLE
Welcome to uor-framework Discussions!

### DISCUSSION THREAD

**Discussion by afflom** [emporous/community#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by afflom** [comment]
Hi I'm Alex. I work at Red Hat as a Solution Architect and am excited to develop UOR with everyone here.

**Comment by christianh814** [comment]
Hi I'm Christian. I work at Red Hat in Technical Marketing, ready to help wherever I can!

**Comment by sabre1041** [comment]
Hey all, I'm Andy, Distinguished Architect from Red Hat and do'er of _many_ things. Looking forward to engaging with the community and moving the UOR initiative forward

**Comment by usrbinkat** [comment]
Hello, I'm Kat Morgan. I am a Sr Developer Advocate at Kong Inc and am super excited to build on this project with you all.
